### DISCUSSION TITLE
huggingface_hub.errors.HFValidationError using LLama3.1-405b

### DISCUSSION THREAD

**Discussion by unrue** [vllm-project/vllm#9657] [General]
Using vllm with LLama3.1-405b, I get the following error:

`huggingface_hub.errors.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name': '****/models/Llama-3.1_405B-Instruct'. Use repo_typeargument if needed.`

The dataset is downloaded from official repo:

[](https://huggingface.co/meta-llama/Llama-3.1-405B-Instruct)

Could you help me to define the problem? Thanks.

**Comment by traa6059a** [comment]
Any answer?

**Comment by Jeiyoon** [comment]
I've encountered the same issue. I just loaded trained model from my disk:

```python
from vllm import LLM

model = LLM(
    model=MODEL_PATH
)
```

huggingface_hub.errors.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name'

**Reply by Jeiyoon** [reply]
> I've encountered the same issue. I just loaded trained model from my disk:
> 
> ```python
> from vllm import LLM
> 
> model = LLM(
>     model=MODEL_PATH
> )
> ```
> 
> huggingface_hub.errors.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name'

Problem has been solved. I changed config file name to "config.json"
